ckanext/versions/logic/dataset_version_action.py

```diff
@@ -116,8 +117,18 @@ def dataset_version_restore(context, data_dict):
     version = version_show(context, data_dict)
     if not version:
         raise toolkit.ObjectNotFound("Version not found")
-    v_name = "restored_{}".format(version['name'])
-    v_notes = "Restored from version: {}".format(version['name'])
+
+    v_name_prefix = _("restored")
+    v_name = "{}_{}".format(v_name_prefix, version['name'])
+    v_notes = _("Restored from version: {}").format(version['name'])
```

Comment:
Just to make sure. You're aware that it's going to create static content localized in the current user language. Hence leading to english ADR to be displaying french text and vice versa.
We have a similar problem with dataset descriptions. Also, I agree that it's most likely that the version name is going to be in french so it's a better idea to keep the language consistent within a single piece of text.

Reply:
Yes so if a country is consistently using the site in French they should not have a problem. However, if you have a mixture of users you will end up with a mixture of English and French strings in the version names.  It's not ideal but I figure it is better than no translation.